dataactvalidator/config/sqlrules/d7_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,8 @@
+-- URI is a required field for aggregate records (i.e., when RecordType = 1)
+SELECT
+    row_number,
```

Comment:
for b12_award_financial_1 we don't return all the rows just the one the rule is specifically checking so I did the same here. No harm in including it though, what do you think?